libs/designsystem/toggle-button/src/toggle-button.component.ts

```diff
@@ -26,5 +33,17 @@ export class ToggleButtonComponent {
 
     this.checked = !this.checked;
     this.checkChanged.emit(this.checked);
+    this.focusToggledButton();
+  }
+
+  focusToggledButton() {
+    // force re-render to ensure that the new button is in the dom
+    this.cdr.detectChanges();
+
+    const buttonToFocus = this.elementRef.nativeElement.querySelector(
+      'button[kirby-button]'
+    ) as HTMLButtonElement;
+
+    buttonToFocus.focus();
```

Comment:
Null check for safety
```suggestion
    buttonToFocus?.focus();
```